### ISSUE TITLE
Autolayout crash with IBGFullActivityIndicatorView

### ISSUE THREAD

**Issue by patbarry** [Instabug/Instabug-iOS#231]
```
Hardware Model:     iPhone7,2
OS Version:         iOS 11.2.2

NSInvalidArgumentException: NSLayoutConstraint for <IBGFullActivityIndicatorView: 0x11558b5c0; frame = (0 0; 0 0); layer = <CALayer: 0x115500380>>: A multiplier of 0 or a nil second item together with a location for the first attribute creates an illegal constraint of a location equal to a constant. Location attributes must be specified in pairs.

0  CoreFoundation    ___exceptionPreprocess
1  libobjc.A.dylib   _objc_exception_throw
2  CoreFoundation    +[NSException raise:format:]
3  Foundation        _ResolveConstraintArguments
4  Foundation        +[NSLayoutConstraint constraintWithItem:attribute:relatedBy:toItem:attribute:multiplier:constant:]
5                    __hidden#5405_ (__hidden#5413_:27:31)
6                    __hidden#4685_ (__hidden#4712_:322:5)
7                    __hidden#4487_ (__hidden#4506_:41:9)
8                    __hidden#1621_ (__hidden#1618_:181:17)
9  libdispatch.dylib __dispatch_call_block_and_release
10 libdispatch.dylib __dispatch_client_callout
11 libdispatch.dylib __dispatch_main_queue_callback_4CF$VARIANT$mp
12 CoreFoundation    ___CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
13 CoreFoundation    ___CFRunLoopRun
14 CoreFoundation    _CFRunLoopRunSpecific
15 GraphicsServices  _GSEventRunModal
16 UIKit             _UIApplicationMain
17                   main (APIRoute+ActionEvents.swift:28:13)
18 libdyld.dylib     _start
```

This crash is happening in v.7.7.1 of the SDK. First time we've seen it in our beta program. Unclear what caused it, but could potentially contact beta user if need be.

**Comment by keith** [comment]
It looks like the `__hidden` symbols here have been introduced recently. With 7.8.1 it looks like the SDK was built with the `-bitcode_hide_symbols` flag. Was this change intentional?

**Comment by Kmohamed** [comment]
@patbarry I will investigate in the crash and get back to you soon

**Comment by Kmohamed** [comment]
@keith actually I notice this issue when we start build our framework using Xcode 9.x 
I tried to pass bitcode_hide_symbols with No but it doesn’t work so we open DTS with apple and still waiting for their response

**Comment by keith** [comment]
@Kmohamed did you just update to 9.x? Since we haven't seen this one until now. Can you make sure that `HIDE_BITCODE_SYMBOLS` and `LD_HIDE_BITCODE_SYMBOLS` are set to `NO`?

**Comment by keith** [comment]
Also, it doesn't look like it, but your `MACH_O_TYPE` isn't set to Relocatable Object File (`mh_object`) right?

**Comment by Kmohamed** [comment]
@keith  I will try to set `LD_HIDE_BITCODE_SYMBOLS` with No.
Related to `MACH_O_TYPE` No it's type is Dynamic library.

**Comment by Kmohamed** [comment]
@keith Actually setting `LD_HIDE_BITCODE_SYMBOLS` `HIDE_BITCODE_SYMBOLS` No works.
But the side effect is the bcSymbolMap is not generated

**Comment by Kmohamed** [comment]
@patbarry this crash is fixed on 7.8 could you please update to the latest.

**Comment by Kmohamed** [comment]
@ptbarry I will close this issue since it's fixed on the latest release also feel free to open it again if you find it.
